seed/data/dev/01000_workload_points_calculations.js

```diff
@@ -43,7 +43,7 @@ exports.seed = function (knex, Promise) {
             nominal_target: 0,
             contracted_hours: 37.5,
             reduction_hours: Math.floor(Math.random() * 6) + 1,
-            cms_reduction_hours: 0.5
+            cms_adjustment_points: 15
           })
```

Comment:
Add some negative points values  to illustrate that they are not considered in the table on the page